core/src/test-integration/java/io/neow3j/protocol/Neow3jIntegrationTest.java

```diff
@@ -1,16 +1,18 @@
 package io.neow3j.protocol;
 
-import static io.neow3j.protocol.TestHelper.NEO_HASH;
 import static org.awaitility.Awaitility.await;
-import static org.hamcrest.Matchers.greaterThanOrEqualTo;
+import static org.hamcrest.Matchers.*;
 
 import io.neow3j.protocol.core.JsonRpc2_0Neow3j;
-import io.neow3j.protocol.core.methods.response.NeoGetBalance;
+import io.neow3j.protocol.core.methods.response.*;
```

Comment:
Same here. Please, check the whole project to see if you changed and added wildcards.